solutions_toolkit/auto_review/auto_review_functions.py

```diff
@@ -1,40 +1,58 @@
+ACCEPTED = "accepted"
```

Comment:
We need to start writing unittests for these functions. I'll post an example one before our next meeting. But it's important that we know these auto review functions are working as expected since they are cross use case.

Reply:
I tested it for MRI and it is working as expected